packages/fyndiq-icon-brand/styles.less

```diff
@@ -65,3 +71,10 @@
     fill: transparent;
   }
 }
+
+.tagline {
+  fill: @red;
+  text-anchor: middle;
+  font-size: 6.5px;
```

Comment:
Having it at 5.5px makes it look more like the logo we display on the monolith (desktop)